cmd/storage-rpc-client.go

```diff
@@ -311,5 +317,5 @@ func (n *networkStorage) RenameFile(srcVolume, srcPath, dstVolume, dstPath strin
 		SrcPath: srcPath,
 		DstVol:  dstVolume,
 		DstPath: dstPath,
-	}, &reply)
+	}, &reply, 10*time.Minute)
```

Comment:
1. Have these timeouts been discussed including side effects of then majorly in write operations?  
2.  These timeouts need to be declared as constants with respective comments.

Reply:
> Have these timeouts been discussed including side effects of then majorly in write operations?

Basing these timeouts based on existing timeouts for Dsync operations, will move them as constants.